packages/services/api/src/modules/support/resolvers/SupportTicket.ts

```diff
@@ -0,0 +1,26 @@
+import { SupportManager } from '../providers/support-manager';
+import type { SupportTicketResolvers } from './../../../__generated__/types.next';
+
+export const SupportTicket: SupportTicketResolvers = {
+  comments: async (ticket, _args, { injector }) => {
+    const response = await injector.get(SupportManager).getTicketComments(ticket.id);
+
+    return {
+      edges: response.nodes.map(comment => ({
+        node: {
+          id: String(comment.id),
+          body: comment.body,
+          createdAt: comment.created_at,
+          fromSupport: comment.fromSupport,
+        },
+        cursor: String(comment.id),
+      })),
+      pageInfo: {
+        endCursor: String(response.nodes[response.nodes.length - 1]?.id ?? ''),
+        hasNextPage: response.meta.has_more,
+        hasPreviousPage: false,
+        startCursor: String(response.nodes[0]?.id ?? ''),
+      },
+    };
+  },
```

Comment:
The schema for this type is this:

```graphql
comments: SupportTicketCommentConnection
```

This seems a bit different from other `*Connection` as they are usually non-nullable i.e. `*Connection!`. 
Looking at the implementation, it can never be null. So I think this could be a way to get around having to declare mapper for `SupportTicket`? 🤔 

Either way, this is part of the migration so we should keep the implementation as-is, however, I think we should change the type to `SupportTicketCommentConnection!` so it correctly reflects the logic. 

Unless I misunderstood it, then please let me know!

Reply:
@kamilkisiela Did you think about anything specific when u implemented this?